ros2nodl/ros2nodl/_verb/_show.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (C) 2020 Canonical Ltd.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License Version 3 as published by
+# the Free Software Foundation.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import pprint
+import shutil
+import sys
+
+import nodl
+from ros2cli.verb import VerbExtension
+from ros2pkg.api import package_name_completer, PackageNotFoundError
+from ros2run.api import ExecutableNameCompleter
+
+
+class _ShowVerb(VerbExtension):
+    """Show NoDL data."""
+
+    def add_arguments(self, parser: argparse.ArgumentParser, cli_name: None = None):
+        # Ignoring type because of https://github.com/python/typeshed/issues/1878
+        parser.add_argument(  # type: ignore
+            'package_name', help='Name of the package to show.'
+        ).completer = package_name_completer
+
+        command_group = parser.add_mutually_exclusive_group(required=True)
+        command_group.add_argument(
+            '-a', '--all', action='store_true', help='Show all executables in package.',
+        )
```

Comment:
I'm second-guessing this arg. What do we expect to happen if we say `ros2 nodl show <package>`? Right now that's an error, because either `-a` or `<executable>` is required. If we get rid of `-a` entirely and make the executable optional, we can default to the `-a` behavior if an executable wasn't specified.

Reply:
I don't like to assume, when run, that it should grab all nodl files, especially given it searches recursively. That being said, that's fairly standard for CLI tools. I'll change it to -a by default